src/systems/filecoin_mining/storage_mining/storage_miner_actor.id

```diff
@@ -136,28 +146,31 @@ type StorageMinerActorCode struct {
     )
     _submitPowerReport(rt Runtime, lastPoSt block.ChainEpoch)
 
-    _verifyPoStSubmission(rt Runtime, postSubmission poster.PoStSubmission) bool
-
     _expirePreCommittedSectors(rt Runtime)
 }
 
 type MinerInfo struct {
+    // The miner actor address assigned by the Storage Power Actor
+    Address                 address.Address
```

Comment:
Blocking: this is really weird, different to all the other actors, and a bad smell. The MinerInfo needs to be known before the miner actor can be created. Why is this needed?

We can add an `ActorAddress()` getter to the VM Runtime if you need access to it in actor code.

Reply:
mmh yeah, I sort of wung it, so thank you for catching. I should have called attention to it. I don't get why this can't be set when CreateMiner is called on the SPA, but it is used to generate randomness in case different miner actors share owner or worker keys...

What do you recommend?

Reply:
I can't find any reference to this field in this PR, but nevertheless do not add it.

If you need to access it from within actor code, add a `ReceiverAddress() addr.Address` accessor method to `Runtime()`. Otherwise, plumb the address through from wherever you looked up the actor state in the state tree.

Reply:
Added in https://github.com/filecoin-project/specs/pull/666

Reply:
Thanks Alex